### DISCUSSION TITLE
What is the latest MP commit to be used with ESP-IDF 4.4?

### DISCUSSION THREAD

**Discussion by MRIIOT** [micropython/micropython#12334] [ESP32] [answered]
What is the latest MP commit to be used with ESP-IDF 4.4?

Prior to e4650125b88a35f074097f16d84a8f49bd22ac06

I'm going to guess 41c91422f083aaf61036c8ead26bf0c43111ae00

**Comment by walklinewang** [comment]
I'm using 7979a4d26776330a89166fa60c917475579f7dd8 with idf v4.4.4 for now, it's worked for me.

**Comment by jimmo** [comment, marked as answer]
41c91422f083aaf61036c8ead26bf0c43111ae00 is the commit before set of commits that moved to IDF 5.x. (See PR #11528)
